engine/session.js

```diff
@@ -85,6 +90,15 @@ class Session {
       if (config.useDemuxedAudio) {
         this.use_demuxed_audio = true;
       }
+      if (config.dummySubtitleEndpoint) {
+        this.dummySubtitleEndpoint = config.dummySubtitleEndpoint;
+      }
+      if (config.subtitleSliceEndpoint) {
+        this.subtitleSliceEndpoint = config.subtitleSliceEndpoint;
+      }
+      if (config.shouldContainSubtitles) {
```

Comment:
change `config.shouldContainSubtitles` to `config.useVTTSubtitles`

Reply:
fixed